src/blogs.twig

```diff
@@ -0,0 +1,30 @@
+{# blogs.html #}
+{% extends "_pages/page_layout.twig" %}
+<head>
+    {% block head %}
+        <link rel="stylesheet" href="css/blogs.css"/>
+    {% endblock %}
```

Comment:
You replaced the block name in the page_layout template but did not do the adjustment here, please replace head with additional_styles